packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -2270,6 +2285,22 @@ class TextSelectionGestureDetectorBuilder {
     final bool isShiftPressedValid = _isShiftPressed && renderEditable.selection?.baseOffset != null;
     switch (defaultTargetPlatform) {
       case TargetPlatform.android:
+        if (editableText.widget.stylusHandwritingEnabled) {
+          final bool stylusEnabled = switch (kind) {
+            PointerDeviceKind.stylus
+            || PointerDeviceKind.invertedStylus =>
+              editableText.widget.stylusHandwritingEnabled,
+            _ => false,
+          };
+          if (stylusEnabled) {
+            Scribe.isFeatureAvailable().then((bool isAvailable) {
+              if (isAvailable) {
+                renderEditable.selectPosition(cause: SelectionChangedCause.tap);
```

Comment:
I think there is a `.scribble` cause?

Reply:
Good call. I've also just opened an issue to rename this to something more generic. https://github.com/flutter/flutter/issues/159223